Gemfile.lock

```diff
@@ -295,4 +295,4 @@ DEPENDENCIES
   webmock (= 1.20.4)
 
 BUNDLED WITH
-   1.10.6
+   1.11.2
```

Comment:
This will probably cause an issue given the current ruby version is deployed with bundler `1.10.6`